hybrid/financial/custom_financial_model.py

```diff
@@ -246,8 +193,8 @@ def execute(self, n=0):
                 rate=self.nominal_discount_rate(
                     inflation_rate=self.value('inflation_rate'),
```

Comment:
is inflation rate supposed to be a ratio or percent?

Reply:
The inflation_rate parameter is a percent in SAM, so nominal_discount_rate is made a percent for consistency with that 'convention'. However, the npv function borrowed from the numpy-financial package takes its rate parameter as a fraction, hence the divide by 100 conversion.

Reply:
is there documentation on the units of the parameters anywhere?